ModManager/ModIoGameInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ModManager.StartupSystem;
+
+namespace ModManager
+{
+    public class ModIoGameInfo : Singleton<ModIoGameInfo>, ILoadable
+    {
+        public static string Url { get; private set; } = null!;
```

Comment:
I don't understand the " = null!;" part, is it needed?

Reply:
Just to hide warnings